scss/components/_activity-card.scss

```diff
@@ -183,3 +184,211 @@
   display: block;
   text-align: center;
 }
+
+/** Designs for big activity card **/
+
+.civicase__activity-card--big {
+  box-shadow: 0 3px 8px 0 rgba(49, 40, 40, 0.15);
+  height: auto;
+  width: 100%;
+
+  .panel-body {
+    display: flex;
+    min-height: 265px;
+    padding: 24px;
+  }
+
+  .panel-body:hover {
+    .civicase__activity-card-big-date::before {
+      border-bottom: 24px solid $gray-lighter;
+      border-right: 24px solid $gray-lighter;
+    }
+  }
+
+  .civicase__contact-card {
+    align-items: center;
+    justify-content: center;
+  }
+
+  .civicase__contact-additional__container--avatar {
+    margin-left: 5px;
+    margin-top: 1px;    
+  }
+
+  .civicase__contact-avatar {
+    margin-top: 0;
+  }
+
+  .civicase__activity-card-row {
+    align-items: flex-start;
+  }
+
+  .civicase__activity-icon-container--ribbon {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    left: 10px;
+    position: absolute;
+    top: 0;
+    width: 40px;
+  }
+
+  .civicase__activity-icon-container--ribbon .civicase__activity-icon {
+    font-size: 22px;
+    left: 0;
+    top: 16px;
+  }
+
+  .civicase__activity-icon-ribbon {
+    border-bottom-width: 10px;
+    border-width: 20px;
+    height: 64px;
+    left: 0;
+  }
+
+  .civicase__activity__tags-container {
+    margin-bottom: 10px;
+  }
+}
+
+.civicase__activity-card--big__inner {
+  padding-left: 50px;
+  padding-right: 230px;
+  position: relative;
+  width: 100%;
+}
+
+.civicase__activity-card-big-date {
```

Comment:
why are there `.civicase__activity-card-big-date`, `.civicase__activity-big-title`, `.civicase__activity-big-subject`, etc. and not just `.civicase__activity-card--big .civicase_activity-card__title` etc?

Also the modifier is not being used correctly (`activity-card-big-date` instead of `activity-card--big__date`)